### PR TITLE
fixes: top options can't work

### PR DESCRIPTION
**Pull request by lifubang**
Signed-off-by: Lifubang <lifubang@acmcoder.com>

### Ⅰ. Describe what this PR did
1. When pouch top --help:
top command is to display the running processes of a container.You can add options just like using Linux ps command.
But when I run: pouch top redisps -aux
It returns:
Error: unknown shorthand flag: 'a' in -aux

2. in https://github.com/alibaba/pouch/blob/master/docs/api/HTTP_API.md
Display the running processes of a container
POST /containers/{id}/top

But in https://github.com/alibaba/pouch/blob/master/client/container_top.go#L19
and https://github.com/alibaba/pouch/blob/master/apis/server/router.go#L57
The type is get.

And, in https://github.com/alibaba/pouch/blob/master/apis/server/container_bridge.go#L323
It uses req.Form.Get

Is post? or get?

### Ⅱ. Does this pull request fix one issue?
NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
Yes, you already had test cases.

### Ⅳ. Describe how to verify it
pouch top redisps -aux

### Ⅴ. Special notes for reviews
Make pouch top options work
API use get method

### CODE REVIEW

**Comment by lifubang** [comment]
So, change to get?

**Comment by allencloud** [comment]
> So, change to get?

I think we need to update the swagger.yml to make method to be `get`. Then @pouchrobot will automatically update the docs without interaction of humans. @lifubang

File: cli/top.go [hunk 1]
```diff
@@ -31,7 +31,8 @@ func (top *TopCommand) Init(c *Cli) {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return top.runTop(args)
 		},
-		Example: topExamples(),
+		Example:            topExamples(),
+		DisableFlagParsing: true,
```
Comment (allencloud) [inline cli/top.go]: Thanks a lot for this update. While for other commands like `create`, `run` and so on, we use `flagSet.SetInterspersed(false)`in 
```
func (cc *CreateCommand) addFlags() {
	flagSet := cc.cmd.Flags()
	flagSet.SetInterspersed(false)

	c := addCommonFlags(flagSet)
	cc.container = c
}
```
How about making pouchd use the unified way of `flagSet.SetInterspersed(false)`?

**Comment by allencloud** [comment]
Do you use wechat or DingTalk? Please add me as a friend with account 15869045568.  Thanks a lot. @lifubang

**Comment by lifubang** [comment]
OK, wait a minite.

File: cli/top.go (hunk 1 above)
Comment (lifubang) [reply cli/top.go]: I have finished.

**Comment by allencloud** [comment]
Thanks again for the fix. While when I was checking the cli integration test, I found that there is no integration test for the args of `pouch top` command. Could you add one to lock down the functionality in test case? @lifubang

**Comment by allencloud** [comment]
Thanks for the test cased added. I feel that we are almost there. Currently there is still some minor issue in code style:
```
#!/bin/bash -eo pipefail
make check
gometalinter
gometalinter --config .gometalinter.json ./...
test/api_container_top_test.go:1::warning: file is not goimported (goimports)
Makefile:148: recipe for target 'gometalinter' failed
make: *** [gometalinter] Error 1
Exited with code 2
```

Is the goimport failing or something esle? @lifubang

**Comment by allencloud** [comment]
LGTM

**Pull request merged**